tldw_Server_API/app/core/Agent_Client_Protocol/adapters/mcp_runners.py

```
@@ -0,0 +1,290 @@
+
+                    await self._emit_event(
+                        AgentEventKind.TOOL_CALL,
+                        {"tool_name": tc.name, "arguments": tc.arguments},
+                        metadata={"_already_approved": True},
+                    )
+
+                    try:
+                        result = await self._transport.call_tool(tc.name, tc.arguments)
+                        output = _extract_text_content(result)
+                        is_error = result.get("isError", False)
+                    except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)